clang/include/clang/Driver/Options.td

```diff
@@ -2357,6 +2357,10 @@ def fsycl_device_code_split_EQ : Joined<["-"], "fsycl-device-code-split=">,
 def fsycl_device_code_split : Flag<["-"], "fsycl-device-code-split">, Alias<fsycl_device_code_split_EQ>,
   AliasArgs<["auto"]>, Flags<[CC1Option, CoreOption]>,
   HelpText<"Perform SYCL device code split in the 'auto' mode, i.e. use heuristic to distribute device code across modules">;
+def fsycl_split_esimd : Flag<["-"], "fsycl-split-esimd">,
```

Comment:
I would suggest to use `fsycl-device-code-split` prefix (e.g. `fsycl-device-code-split-esimd`) for greps to find both options related to the device code split.

Reply:
OK, will change.

Reply:
Fixed.